install_config/web_console_customization.adoc

```diff
@@ -265,6 +265,10 @@ oauthConfig:
 Relative paths are resolved relative to the master configuration file. You must
 restart the server after changing this configuration.
 
+Each time a user's token to {product-title} expires, the user is presented with
```

Comment:
also this is only true if either (1) there are multiple login providers configured or (2) the alwaysShowProviderSelection option in master-config.yaml is set to true.